docs/src/guide/parallelReduce/index.md

```diff
@@ -177,33 +196,41 @@ load: /examples/point-of-sale/index.rsh
 md5: 2d9202924ee606ab47726038fbde05a3
 range: 19-24
 ```
-- Line 19 starts a local step for `{!rsh} A`.
+- Line 19 starts a local step for `A`.
 - Line 20 unpacks values from the frontend `params` object.
 - Line 22 `{!rsh} publish`es these values to the blockchain.
 
 :::note
-We cannot attach the `{!rsh} A.pay([[supply, tok]])` to the first `{!rsh} publish` of the DApp. This is because the Smart Contract doesn't yet know of our token. We first need to complete a `{!rsh} publish` of the token ID.
+We cannot attach the `A.pay([[supply, tok]])` to the first `{!rsh} publish` of the DApp. This is because the Smart Contract doesn't yet know of our token. We first need to complete a `{!rsh} publish` of the token ID.
 
 Reach supports network tokens by default, which is why this constraint does not exist when paying network tokens.
 :::
 
-You may have noticed the syntactic Tuple in the `{!rsh} .pay`. Token IDs and the amounts to be paid need to be specified in Tuples. The first argument is the default network token and is synthesized for you, just specify the quantity -- if you leave the quantity out it will be synthesized to zero for you.
+You may have noticed the syntactic Tuple in the `{!rsh} .pay`. 
+
+Token IDs and the amounts to be paid need to be specified in `{!rsh} Tuples`. 
+
+The first argument is the default network token and is synthesized for you, just specify the quantity -- if you leave the quantity out it will be synthesized to zero for you.
+
+This means `A.pay([[supply, tok]])` is equal to `A.pay([0, [supply, tok]])`. Where zero specifies the number of network tokens. 
+
+This syntax will come up again.
 
-This means `{!rsh} A.pay([[supply, tok]])` is equal to `{!rsh} A.pay([0, [supply, tok]])`. Where zero specifies the number of network tokens. This syntax will come up again.
+This particular program sells these as loyalty tokens, but also allows the user to return the token in the event of a refund. 
 
-This particular program sells these as loyalty tokens, but also allows the user to return the token in the event of a refund. This means that our `{!rsh} parallelReduce` will need to *accept* these non-network tokens as payment in our API member function.
+This means that our `{!rsh} parallelReduce` will need to *accept* these non-network tokens as payment in our API member function.
 ```
 load: /examples/point-of-sale/index.rsh
 md5: 2d9202924ee606ab47726038fbde05a3
 range: 28-33
 ```
 - Line 28 starts our `{!rsh} parallelReduce`.
-- Line 29 is new. `{!rsh} .paySpec([tok])` tells our `{!rsh} parallelReduce` that it can also accept payment in this non-network token.
+- Line 29 is new. `.paySpec([tok])` tells our `{!rsh} parallelReduce` that it can also accept payment in this non-network token.
```

Comment:
```suggestion
- Line 29 is new. `{!rsh} .paySpec([tok])` tells our `{!rsh} parallelReduce` that it can also accept payment in this non-network token.
```